### PR TITLE
Update history page to use resources

### PR DESCRIPTION
**Pull request by lhohenrique**

### CODE REVIEW

**Review by gmojunior — APPROVED** [review]

File: PontoFacil/PontoFacil/Views/HistoryPage.xaml [hunk 1]
```diff
@@ -62,52 +62,51 @@
...
+                    <TextBlock x:Uid="StartTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="EndTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="StartTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="EndTime" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="WorkedHours" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="OvertimeHours" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="TotalHours" Style="{StaticResource HistoryTitle}"></TextBlock>
+                    <TextBlock x:Uid="Waiver" Style="{StaticResource HistoryTitle}"></TextBlock>
                 </StackPanel>
                 <ListView x:Name="historyList" Margin="0,20,0,0" Height="auto" MaxHeight="700" ScrollViewer.VerticalScrollBarVisibility="auto"
```
Comment (thiago-rocha) [inline PontoFacil/PontoFacil/Views/HistoryPage.xaml]: Extract the style of this ListView to the Page Resouce.

**Review by soutolucas — APPROVED** [review]

**Review by thiago-rocha — COMMENTED** [review]
@lhohenrique I add just one comment but it will be nice with you go through all xaml component checking if they should be extract yo the style resource.

File: PontoFacil/PontoFacil/Views/HistoryPage.xaml (hunk 1 above)
Comment (lhohenrique) [reply PontoFacil/PontoFacil/Views/HistoryPage.xaml]: Done @thiago-rocha ...Ready to review.

**Review by thiago-rocha — APPROVED** [review]

**Comment by thiago-rocha** [comment]
Rebase conflict, please resend

**Pull request merged**